### ISSUE TITLE
BUG: French Add User not working

### ISSUE THREAD

**Issue by ecadieux** [orangehrm/orangehrm#452]
French Add User not working

> Uncaught SyntaxError: Unexpected identifier
> systemUserSuccess.js:11 Uncaught ReferenceError: isUniqueUserUrl is not defined
>     at HTMLDocument.<anonymous> (systemUserSuccess.js:11)
>     at o (jquery-1.7.2.min.js:2)
>     at Object.fireWith (jquery-1.7.2.min.js:2)
>     at Function.ready (jquery-1.7.2.min.js:2)
>     at HTMLDocument.B (jquery-1.7.2.min.js:2)

This issue comes from line 47 in symfony/plugins/orangehrmAdminPlugin/modules/admin/templates/saveSystemUserSuccess.php:

``` 
var user_ValidEmployee          = '<?php echo __(ValidationMessages::EMPLOYEE_DOES_NOT_EXIST); ?>';
``` 

Which in French gives this:

``` 
var user_ValidEmployee          = 'L'employé n'existe pas'; 
```

Replacing single quotes by double quotes fixes the issue:
``` 
var user_ValidEmployee          = "<?php echo __(ValidationMessages::EMPLOYEE_DOES_NOT_EXIST); ?>";
```

**Comment by samanthajayasinghe** [comment]
Thanks @ecadieux, We will fix the issue in next version

**Comment by icechathuranga** [comment]
you can use esc_js_no_entities function to overcome this issue. 

var user_ValidEmployee          = '<?php echo esc_js_no_entities ( __(ValidationMessages::EMPLOYEE_DOES_NOT_EXIST)) ; ?>';

Check the other places also.

**Comment by icechathuranga** [comment]
you can use esc_js_no_entities function to overcome this issue.

ex: 
var user_ValidEmployee          = '<?php echo esc_js_no_entities(__(ValidationMessages::EMPLOYEE_DOES_NOT_EXIST)); ?>';

**Comment by icechathuranga** [comment]
you can use esc_js_no_entities function to overcome this issue.
var user_ValidEmployee          = '<?php echo esc_js_no_entities(__(ValidationMessages::EMPLOYEE_DOES_NOT_EXIST)); ?>';

**Comment by icechathuranga** [comment]
Use esc_js_no_entities function to fix this issue.

**Comment by samanthajayasinghe** [comment]
@ecadieux this issue was fixed in this PR https://github.com/orangehrm/orangehrm/pull/456 and merged into next release

**Comment by kavithagurusamy** [comment]
This is fixed in 4.3.1
The latest release is available on SourceForge
https://sourceforge.net/projects/orangehrm/files/stable/4.3.1/
